src/steps/run_scripts.ts

```diff
@@ -29,10 +29,13 @@ const runShellScript = (cmd, cwd = __dirname, options = {}) => {
 
 export default async (options: Map<string, any>) => {
   const dest = options.get("dest");
+  // For better performance, prefer yarn over npm if available
+  const yarnInstalled = !!exec("yarn --version", { silent: true }).stdout;
+  const pkgManager = yarnInstalled ? "yarn" : "npm";
   console.log(chalk.bold.green("Installing dependencies"));
-  await runShellScript("yarn install --force", dest);
+  await runShellScript(`${pkgManager} install --force`, dest);
```

Comment:
We may not need `--force` anymore.  I added it when still working on using npm's offline cache feature.  In that case, sass' native bindings weren't being built.  The helper no longer uses offline cache, so if you're interested, try without the `--force`